src/models/Production.ts

```diff
@@ -0,0 +1,57 @@
+import { Model } from "objection";
+import { ProductionStatus } from "./ProductionStatus";
+import { Item } from "./Item";
+import { ProductionHistory } from "./ProductionHistory";
+
+export class Production extends Model {
+  production_guid: string;
+  production_status_guid: string;
+  finished_item_guid: string;
+  args: object;
+
+  static tableName = 'productions';
+  static idColumn = 'production_guid';
+  
+  static jsonSchema = {
+    type: 'object',
+    required: ['production_status_guid', 'finished_item_guid', 'args'],
+
+    properties: {
+      production_guid: { type: "string" },
+      production_status_guid: { type: "string" },
+      finished_item_guid: { type: "string" },
+      args: { 
+        type: "object", 
+        properties: {
+          additionalProperties: true
+        } 
+      },
+    },
+  }
+  static relationMappings = {
```

Comment:
when returning the production model, we should have possibility to return the relationship based on include query strings

Reply:
Will be done in new ticket